propagators/opentelemetry-propagator-grpc-census-binary/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@opentelemetry/propagator-grpc",
```

Comment:
Name should also be changed here

Reply:
Thanks. The package.json change (as well as README updates) should have been in the previous commit. Latest commit should resolve.